### PR TITLE
TG-61: Redux setup

### PR DESCRIPTION
**Pull request by camilavaleriarojas**
Implementation of redux, redux-thunk, redux-devtools-extension, and creation of store and the folders to each entity.

### CODE REVIEW

File: src/redux/rootReducer.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import tasksReducer from '../redux/task/reducer';
+import timesheetsReducer from '../redux/timesheets/reducer';
+
+const rootReducer = combineReducers({
+  admins: adminsReducer,
+  employees: employeesReducer,
+  projects: projectsReducer,
+  superAdmins: superAdminsReducer,
+  tasks: tasksReducer,
+  timesheets: timesheetsReducer
```
Comment (baudraccoverona) [inline src/redux/rootReducer.js:15]:
```suggestion
  timeSheets: timeSheetsReducer
```

File: src/redux/rootReducer.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { combineReducers } from 'redux';
+import adminsReducer from '../redux/admins/reducer';
+import employeesReducer from '../redux/employees/reducer';
+import projectsReducer from '../redux/projects/reducer';
+import superAdminsReducer from '../redux/superAdmins/reducer';
+import tasksReducer from '../redux/task/reducer';
+import timesheetsReducer from '../redux/timesheets/reducer';
```
[the remark this answers is not in the document]
Comment (baudraccoverona) [reply src/redux/rootReducer.js:7]: and also change the timeSheets folder name to use camelCase please 🙏

[the remark this answers is not in the document]
Comment (camilavaleriarojas) [reply src/redux/rootReducer.js:7]: Sorry T forgot it!

[the remark this answers is not in the document]
Comment (camilavaleriarojas) [reply src/redux/rootReducer.js:7]: Sorry I forgot it!

**Review by baudraccoverona — APPROVED** [review]

**Pull request merged**